modules/integrations/arrow-integrations-retrofit-adapter/src/main/kotlin/arrow.integrations.retrofit.adapter/Async2CallAdapterFactory.kt

```diff
@@ -0,0 +1,38 @@
+package arrow.integrations.retrofit.adapter
```

Comment:
For what I can see in the file name this is in a folder with `.` but each one of the sub packages should be on its own folder